composer.json

```diff
@@ -36,7 +36,7 @@
         "issues": "https://github.com/zendframework/zend-expressive-skeleton/issues",
         "source": "https://github.com/zendframework/zend-expressive-skeleton",
         "rss": "https://github.com/zendframework/zend-expressive-skeleton/releases.atom",
-        "slack": "https://zendframework-slack.herokuapp.com",
+        "chat": "https://zendframework-slack.herokuapp.com",
```

Comment:
Is `slack` or `chat` allowed?

See: https://github.com/composer/composer/blob/4d8b9be5b6a0bda0a3d24c0a6ff89d7ccb26f715/res/composer-schema.json#L461

Reply:
Good point and I'm not sure. We can always call it irc, because it's just irc 2.0 :)

Reply:
Schema pretty clearly only shows "irc". I'll make that change on merge.